src/components/Buttons/Button.js

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Button.propTypes = {
+  disabled: PropTypes.bool.isRequired,
+  handleClick: PropTypes.func,
+  children: PropTypes.string.isRequired,
+}
+
+export default function Button({ disabled, handleClick, children }) {
+  return (
+    <ButtonStyled
+      disabled={disabled}
+      onClick={handleClick}
+      data-testid="button"
```

Comment:
Do you use that data-testid anywhere in your tests right now? Couldn`t find it. Otherwise maybe just leave it out for now.

Reply:
same here :)